Second part of Portfolio

tchasinga opened the pull request:
_NB : Run this project to the size device of 375px as Instructions give by Microverse thank you_ 

Additional Comment 

Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
Linter should be set up correctly for this repository.
Linter result should be green.
There should be no custom changes to linter config.
Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
README file should be descriptive.
Commit messages should be meaningful.
Pull Request should have a descriptive title telling you what changes are inside.
Pull Request should have a short summary describing introduced changes in more detail.
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
Branch should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
Pull Request is made from 'feature branch' into the main branch.

tchasinga committed:
Updating area of code

in this I've performed all of the Important requirements

tchasinga committed:
Code note and Update

TadesseDev commented on index.html:
```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
+        HERE I'LL USE SESSION TAG TO CODE CARD , AND REMEMBER AGAIN YOU MUST REALLY RUN THIS CODE TO 375PX DEVICE 
+        THANK YOU ..... 
+    -->
+    <section class="card-id">
+      <div class="card-one">
+        <img src="Icon and img/Tonic Portfolio.png" alt="" />
```
- [ ] Dear @tchasinga, for your work section, Please make sure to align your contents to the center of the page, Besides that, notice the text for the `See Project` should be centered. This is to make sure you have a design that is identical to the figma design.

TadesseDev commented on index.html:
```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
+      </div>
+    </section>
+
+    <footer>
+      <div class="aboutme">
+        <h1>About Myself</h1>
```
- [ ] Dear @tchasinga, Please notice this section is a different section from the footer and should be wrapped in its own `section` tag instead of the footer. Please Update your `about me` section to be in its own separate section.

TadesseDev commented on index.html:
```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
+      </div>
+      <div class="contactLink">
+        <ul>
+          <li>
+            <a href="https://twitter.com/TchasingaJacque"></a>
+            <img src="img/Twitter.png" alt="" />
+          </li>
```
- [ ] Dear @tchasinga, Here we need to see every icon as a link, Hence every icon should be wrapped inside the `a` tag. That will make your icon a link rather than just another image. Please wrap your icons in the `a` tag. 
> something like ...
```html 
 <li>
    <a href="https://twitter.com/TchasingaJacque"><img src="img/Twitter.png" alt="" /></a>
  </li>
```

TadesseDev commented on index.html:
```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
+            <a href="https://wellfound.com/jobs"
+              ><img src="img/welldone.png" alt=""
+            /></a>
+          </li>
         </ul>
+        <h3 class="resum">Get my resum</h3>
       </div>
```
- [ ] For the `Get my resume` text content, please remember that this is going to be used as a link to open a link to your resume. Hence we need to make it a link(`a`)/button(`button`) tag instead of an `h3` tag.

TadesseDev commented on style.css:
```diff
@@ -83,4 +85,277 @@
+    color: white;
+  }
+
+  footer {
+    position: absolute;
+    width: 375px;
+    height: 1263px;
+    background: #fff;
+    border-top-right-radius: 120px;
+  }
+
+  .aboutme {
+    margin-top: 100px;
+  }
```
- [ ] Dear @tchasinga, as part of the main requirements for the project the content of the `about me` section is expected to be aligned with Flexbox. This is to make sure you master the skill of using different alignments in your website.

TadesseDev reviewed (changes requested):
### Hi @tchasinga 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7](https://user-images.githubusercontent.com/69077061/212385281-4797c58a-d358-423a-aa7c-832c22203865.gif)




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tchasinga committed:
Requirement done

IN THIS CORRECTION I'VE DONE AND MAKE SURE THAT ALL OF REQUIREMENT GIVEN
        BY MICROVERSE ARE DONE , AND I'VE DONE IT IN A GOOD WAY , I'VE USED SECTION TAG TO CODE CARD
        PLEASE FOR THE NEXT CODE VIEWER HELP ME TO MOVE FORWARD , I'VE ONLY TRHEE ATTEMPS, BE AT MY PLACE ..,

IngDavidTM reviewed (approved):
Hi @tchasinga 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
